### DISCUSSION TITLE
Turning Cog - M3 Won't Start! [SOLVED]

### DISCUSSION THREAD

**Discussion by Chipperchap** [sircharlo/meeting-media-manager#400] [General]
Hi all,
PANIC! - M3 won't start, it just shows a continuous turning cog in the start process - internet connection is fine - any ideas please?

**Comment by Chipperchap** [comment]
Update: I right clicked on the desktop icon and selected "Run as Administrator" and now it gets to the checkered flag part, but is stuck at # 5 seconds in the countdown start phase...

**Reply by sircharlo** [reply]
Try deleting the M³ folder from the appdata directory. What OS are you running?

**Reply by sircharlo** [reply]
If Windows, delete both these folders if they exist:

    C:\Users\<username>\AppData\Roaming\jw-meeting-media-fetcher
    C:\Users\<username>\AppData\Roaming\meeting-media-manager

**Comment by Chipperchap** [comment]
Sorry for the delay - I was called out to work.  
I was at another Kingdom Hall helping set things up for them and found the issue as described above. Now I am home and checked my PC (Windows 10) and the issue is identical.  Sometimes it doesn't get past the cog, and other times it gets to the flag part with five seconds left to count and hangs. 
Please see two .gif videos as below: 
[Please note each video is sped up 8x to avoid boring the viewer - just to illustrate that nothing happens.]

![Cog Only (8x Speed)](https://user-images.githubusercontent.com/35364657/175464887-eab356d0-7f4f-48c7-a352-d551b9b4368f.gif)

![Cog to Flag (8x Speed)](https://user-images.githubusercontent.com/35364657/175464900-0af0861e-9f18-45a0-81d2-d9461babb78f.gif)

I tried deleting the files mentioned as in the reply post (for which I am very grateful) but that was a minor nightmare in having to set M3 up as if a new user again several times.  BTW is there a way to save and load my personalised setting preferences?

UPDATE:  I just logged in remotely to our K-Hall and M3 is doing the same there.  It seems to be universal (all W10 PC's).

Don't know if this helps...:

{
  "autoOpenFolderWhenDone": null,
  "autoQuitWhenDone": null,
  "autoRunAtBoot": null,
  "autoStartSync": true,
  "congServer": "my.powerfolder.com",
  "congServerDir": "/webdav/JWMMF",
  "congServerPass": "XXXX",
  "congServerPort": 443,
  "congServerUser": "xxxxx@outlook.com",
  "congregationName": "[Name]",
  "disableHardwareAcceleration": null,
  "enableMediaDisplayButton": true,
  "enableMp4Conversion": null,
  "enableMusicButton": true,
  "enableMusicFadeOut": true,
  "enableObs": null,
  "enableVlcPlaylistCreation": null,
  "excludeLffi": true,
  "excludeLffiImages": true,
  "excludeTh": true,
  "hideMediaLogo": null,
  "keepOriginalsAfterConversion": true,
  "lang": "E",
  "langUpdatedLast": "2022-06-15T09:09:34.042Z",
  "localAdditionalMediaPrompt": null,
  "localAppLang": "en",
  "localOutputPath": "C:\\Users\\simon\\Documents\\JWMMF Master",
  "maxRes": "720p",
  "musicFadeOutTime": "5",
  "musicFadeOutType": "smart",
  "musicVolume": "10",
  "mwDay": "3",
  "mwStartTime": "07:00",
  "obsCameraScene": "Cam Only",
  "obsMediaScene": "MMM",
  "obsPassword": "Changeme",
  "obsPort": "4444",
  "outputFolderDateFormat": "YYYY-MM-DD",
  "preferredOutput": "917481049",
  "weDay": "6",
  "weStartTime": "10:00"
}

Kind regards
Simon

**Comment by Chipperchap** [comment]
UPDATE: The issue appears to be related to the congregation sync / cloud folder, "PowerFolder". It appears that their servers have been offline and nothing is able to sync. 
I waited about an hour and their cloud website is up and running again but the folders cannot be opened. M3 keeps coming up with an error when attempting to "Update Media Folders" that the "Hidden" and "Media" folders "directory" cannot be read. 

![Directory](https://user-images.githubusercontent.com/35364657/175482906-f41d2320-f77d-4051-94a2-816aa85c0153.png)

I cleared the cache, and deleted the AppData MMM folder and reloaded everything.  

I will wait a little longer and see if "PowerFolder" will be fixed... at least in the meantime M3 is actually loading and I am able to view and display all JW.ORG content... just no content previously uploaded to to "PowerFolder". 

Curious - Are the other cloud folders suggested by M3 any easier to use / more reliable?

UPDATE: I remembered that I had, on an earlier occasion, made a "4Shared" account - so loaded those credentials into M3 and M3 works flawlessly!  So that seems to confirm that it is "PowerFolder" that is at issue.

**Reply by sircharlo** [reply]
I've added an additional bit of error catching which should prevent that endless "wait" time when a remote server appears online but just doesn't respond properly and alert the user.
